agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java

```diff
@@ -725,55 +717,81 @@ static Configuration getConfigurationFromEnvVar(String content) {
     return configuration;
   }
 
+  private static Configuration loadJsonConfigFile(Path configPath) {
+    if (!Files.exists(configPath)) {
+      throw new IllegalStateException("config file does not exist: " + configPath);
```

Comment:
If this `IllegalStateException` and the following one are not catched, it may be more explicit for our users to have an Application Insights exception, for example `AiJsonFileNotFoundException`